### PR TITLE
Install page

### PR DESCRIPTION
**Pull request by nl0**
## TODO

- [x] proper links: contact email, prereq, marketplace

## Questions
- [x] keep website theme (dark bg) or switch to app theme (light bg)?
- [x] should we add a link to this page to header and/or footer navigation?
- [x] should we show this page in all modes?

install-page

### CODE REVIEW

File: catalog/app/website/pages/Install/Install.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      </p>
+      <ol className={classes.list}>
+        <li>
+          Ensure you have <M.Link href={PREREQ_LINK}>the prerequisites</M.Link> ready. Of
+          note you will need an AWS account with Administrator access, an S3 bucket, and
+          the ability to create DNS entries for your preferred domain (e.g.{' '}
+          <code>your-company.com</code>).
+        </li>
+        <li>
+          <kbd>pip install quilt-admin</kbd>
```
Comment (armandmcqueen) [inline catalog/app/website/pages/Install/Install.js]: Now called `quilt-stack-installer`

File: catalog/app/website/pages/Install/Install.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        </li>
+        <li>
+          Visit <M.Link href={MP_LINK}>Quilt Business</M.Link> on AWS Marketplace and
+          click Continue to Subsribe. You must complete this step in order for your
+          installation to function properly. This step creates a license key for you.
```
Comment (armandmcqueen) [inline catalog/app/website/pages/Install/Install.js]: > This step creates a license key for you.

I think this phrasing might be confusing. I think users will expect it to hand you a license key, but it does it quietly in the background. Maybe "After completing this step, you will not need a license key". Or maybe just remove that sentence.

File: catalog/app/website/pages/Install/Install.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          <div className={classes.marketplace}>
+            <img src={mp} alt="" />
+          </div>
+        </li>
+        <li>
+          <kbd>quilt-admin install</kbd>
```
Comment (armandmcqueen) [inline catalog/app/website/pages/Install/Install.js]: `quilt-stack installer` install

File: catalog/app/website/pages/Install/Install.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        </li>
+      </ol>
+      <p className={classes.p}>
+        The <code>quilt-admin</code> utility will walk you through the installation. An
```
Comment (armandmcqueen) [inline catalog/app/website/pages/Install/Install.js]: `quilt-stack-installer`

File: catalog/app/website/pages/Install/Install.js (hunk 3 above)
Comment (nl0) [reply catalog/app/website/pages/Install/Install.js]:
```suggestion
          <kbd>quilt-stack-installer install</kbd>
```

File: catalog/app/website/pages/Install/Install.js (hunk 1 above)
Comment (nl0) [reply catalog/app/website/pages/Install/Install.js]:
```suggestion
          <kbd>pip install quilt-stack-installer</kbd>
```

File: catalog/app/website/pages/Install/Install.js (hunk 4 above)
Comment (nl0) [reply catalog/app/website/pages/Install/Install.js]:
```suggestion
        The <code>quilt-stack-installer</code> utility will walk you through the installation. An
```

File: catalog/app/website/pages/Install/Install.js (hunk 2 above)
Comment (nl0) [reply catalog/app/website/pages/Install/Install.js]: @akarve which wording should we use here?

Comment (akarve) [reply catalog/app/website/pages/Install/Install.js]: OK to eliminate the last sentence altogether ~This step ...~

File: catalog/app/website/pages/Install/Install.js [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+    return <M.Link {...props}>{children}</M.Link>
+  }
+  return (
+    <M.Container className={classes.container}>
+      <M.Box pt={5}>
+        <M.Typography variant="h1">Install a private Quilt instance</M.Typography>
+      </M.Box>
+      <p className={classes.p}>
+        The following instructions will guide you through the process of installing Quilt
+        on your own VPS on AWS. If you have any questions, {contactLink('contact us')}.
```
Comment (armandmcqueen) [inline catalog/app/website/pages/Install/Install.js]:
```suggestion
        on your own VPC on AWS. If you have any questions, {contactLink('contact us')}.
```

**Pull request merged**